pkg/util/helm/values.go

```diff
@@ -31,10 +31,10 @@ func FetchImageFromHelmValues(clientSet *kubernetes.Clientset, namespace, name s
 		KubeClient:       kube.New(restClientGetter),
 	})
 	getValues.AllValues = true
-	helmValues, err := getValues.Run("harvester")
+	helmValues, err := getValues.Run(name)
```

Comment:
cc @FrankYang0529 The given chart is used to get values. With the update, the `rancher-logging` chart also gets wanted image lists.